templates/prescription/general_edit.html.twig

```diff
@@ -91,195 +89,244 @@
                 <input type="checkbox" id="checked_formulary_flag" name="checked_formulary_flag" />
-            {else}
+            {% else %}
                 <input type="checkbox" id="checked_formulary_flag" name="checked_formulary_flag" checked="checked" />
-            {/if}
-            <span class="text">{xlt t='Checked Drug Formulary?'}</span><br />
+            {% endif %}
+            <span class="text">{{'Checked Drug Formulary?'|xlt}}</span><br />
 
-            {amcCollect amc_id='e_prescribe_cont_subst_amc' patient_id=$prescription->patient->id object_category='prescriptions' object_id=$prescription->id}
-            {if not $amcCollectReturn}
+            {% if amcCollectReturnControlledSubstances is not empty %}
                 <input type="checkbox" id="controlled_substance_flag" name="controlled_substance_flag" />
-            {else}
+            {% else %}
                 <input type="checkbox" id="controlled_substance_flag" name="controlled_substance_flag" checked="checked" />
-            {/if}
-            <span class="text">{xlt t='Controlled Substance?'}</span><br />
+            {% endif %}
+            <span class="text">{{'Controlled Substance?'|xlt}}</span><br />
             </div>
         </div>
-        {/if}
+        {% endif %}
 
         <div class="form-group mt-3">
-            <label>{xlt t='Currently Active'}</label>
-            <input class="input-sm" type="checkbox" name="active" value="1"{if $prescription->get_active() > 0} checked{/if} />
+            <label>{{'Currently Active'|xlt}}</label>
+            <input class="input-sm" type="checkbox" name="active" value="1"{% if prescription.get_active() > 0 %} checked{% endif %} />
         </div>
         <div class="form-group mt-3">
-            <label>{xlt t='Starting Date'}</label>
-            <input type="text" size="10" class="datepicker form-control" name="start_date" id="start_date" value="{$prescription->start_date|oeFormatShortDate|attr}" />
+            <label>{{'Starting Date'|xlt}}</label>
+            <input type="text" size="10" class="datepicker form-control" name="start_date" id="start_date" value="{{prescription.start_date|shortDate|attr}}" />
         </div>
         <div class="form-group mt-3">
-            <label>{xlt t='Provider'}</label>
-            {html_options class="input-sm form-control" name="provider_id" options=$prescription->provider->utility_provider_array() selected=$prescription->provider->get_id()}
-            <input type="hidden" name="patient_id" value="{$prescription->patient->id|attr}" />
+            <label>{{'Provider'|xlt}}</label>
+            <select class="input sm form-control" name="provider_id">
+                {% for providerId, providerName in prescription.provider.utility_provider_array() %}
+                    <option value="{{ provider.id }}" {% if providerId == prescription.provider.get_id() %} selected="selected" {% endif %}>{{ providerName|text }}</option>
```

Comment:
attr after provider.id

Reply:
Thanks @bradymiller always appreciate you taking another eyeball at these things.